<?xml version="1.0"?>
<launch>
    
    <arg name="trans_list" default="None" />
    <arg name="path" default="/home/mikulas/crazyflie_ws/src/crazyflie_ros/crazyflie_demo/src/" />
    <arg name="n_swarm" default="1" /> 
    
    <group ns="crazyflie1">
        <node pkg="crazyflie_demo" type="error_republisher.py" name="error_republisher" output="screen">
            <param name="n_agent" value="0" />
            <param name="sensor_topic" value="/crazyflie1/vrpn_client_node/crazyflie1/pose" />
            <param name="reference_topic" value="/crazyflie1/cmd_full_state" />
            <param name="frame_id" value="/world" />
            <param name="n_swarm" value="$(arg n_swarm)" />
        </node>
    </group>
      
    <!--group ns="crazyflie2">
    </group-->
        
    <!-- run rviz
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find crazyflie_demo)/launch/crazyflie_pos_multi.rviz" required="true" />
    -->
    
 </launch>
